functions-homework.js

```diff
@@ -0,0 +1,46 @@
+// 1
+function getGreetings(name) {
+  alert('Hi, ' + name + ' What`s up?');
+}
+//getGreetings('kuba klin');
+
+// 2
+function isThisMyName(personName) {
```

Comment:
Let's use the knowledge from the lesson about logical operators. You could solve this function in an elegant way in just one line with a single return statement.